exporter/sumologicexporter/config.go

```diff
@@ -46,6 +51,11 @@ type Config struct {
 	// Decompose OTLP Histograms into individual metrics, similar to how they're represented in Prometheus format
 	DecomposeOtlpHistograms bool `mapstructure:"decompose_otlp_histograms"`
 
+	// Traces related configuration
+	// The format of traces you will be sending, currently only otlp format is supported
+	TraceFormat TraceFormatType `mapstructure:"trace_format"`
```

Comment:
Do you want to add a README entry for this option? 

Side note, if there's only one valid, is it worth having a config option? (I guess this depends on whether there are plans to support more formats, I'll defer to you and other code owners here as to usefulness.)

Reply:
I just removed it. It seems we do not use it anywhere, so should be safe to remove. We can add it in the future if we need to